doc/reference/reference_lua/errcodes.rst

```diff
@@ -1,112 +1,136 @@
-    |                   | write-ahead log. Some sort of disk error.              |
-    +-------------------+--------------------------------------------------------+
-    | ER_KEY_PART_COUNT | Key part count is not the same as                      |
-    |                   | index part count                                       |
-    +-------------------+--------------------------------------------------------+
-    | ER_NO_SUCH_SPACE  | The specified space does not exist.                    |
-    |                   |                                                        |
-    +-------------------+--------------------------------------------------------+
-    | ER_NO_SUCH_INDEX  | The specified index in the specified                   |
-    |                   | space does not exist.                                  |
-    +-------------------+--------------------------------------------------------+
-    | ER_PROC_LUA       | An error occurred inside a Lua procedure.              |
-    |                   |                                                        |
-    +-------------------+--------------------------------------------------------+
-    | ER_FIBER_STACK    | The recursion limit was reached when                   |
-    |                   | creating a new fiber. This usually                     |
-    |                   | indicates that a stored procedure is                   |
-    |                   | recursively invoking itself too often.                 |
-    +-------------------+--------------------------------------------------------+
-    | ER_UPDATE_FIELD   | An error occurred during update of a                   |
-    |                   | field.                                                 |
-    +-------------------+--------------------------------------------------------+
-    | ER_TUPLE_FOUND    | A duplicate key exists in a unique                     |
-    |                   | index.                                                 |
-    +-------------------+--------------------------------------------------------+
+    ..  rst-class:: left-align-column-1
+    ..  rst-class:: left-align-column-2
+
+    ..  list-table::
+        :widths: 25 35 40
+        :header-rows: 1
+
+        *   - Code
+            - box.error value
+            - Description
+
+        *   - ER_NONMASTER
+            - box.error.NONMASTER
+            - (In replication) A server instance cannot modify data unless it is a master.
+
+        *   - ER_ILLEGAL_PARAMS
+            - box.error.ILLEGAL_PARAMS
+            - Illegal parameters. Malformed protocol message.
+
+        *   - ER_MEMORY_ISSUE
+            - box.error.MEMORY_ISSUE
+            - Out of memory: :ref:`memtx_memory <cfg_storage-memtx_memory>` limit has been reached.
+
+        *   - ER_WAL_IO
+            - box.error.WAL_IO
+            - Failed to write to disk. May mean: failed  to record a change in the write-ahead log.
+
+        *   - ER_READONLY
```

Comment:
Maybe add a link to descriptions of specific parameters in error_object reference?